tools/build_pytorch_libs.sh

```diff
@@ -299,7 +289,6 @@ function build_caffe2() {
 
   # Fix rpaths of shared libraries
   if [[ $(uname) == 'Darwin' ]]; then
-      # root/torch/lib/tmp_install/lib
       report "Updating all install_names in $INSTALL_DIR/lib"
       pushd "$INSTALL_DIR/lib"
       for lib in *.dylib; do
```

Comment:
Now that we are not copying the libraries, do we still need to update rpath?  I do not know, but this is a major source of wasted time in the incremental build on osx.

Reply:
I tried deleting this in a separate PR and nothing in CI broke, but @pjh5 was concerned it might break things that aren't checked by CI, so I'm holding off for now